apps/website/.stylelintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ["configs/.stylelintrc.json"],
```

Comment:
Is this path correct? What are these paths relative to? If it if relative to the current folder, then `apps/website/configs/.stylelintrc.json` doesn't exist?

Reply:
`configs` is the name of the `internal/configs` workspace.

check other `.stylelinrc.json` files